Pages/product.html

```diff
@@ -0,0 +1,851 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://orcaui.netlify.app/styles/main.css" />
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/Styles/product.css">
+    <link rel="shortcut icon" href="/Assets/favicon.svg" type="image/x-icon">
+    <meta name="author" content="Hitesh Verma">
+    <meta name="description" content="Shop from the best collection of motorsport posters.">
+    <title>Products | LightsOut</title>
+</head>
+
+<body>
+    <nav class="w-100vw flex-space_between-center px-10 py-1 bg-white">
+        <a href="/index.html">
+            <h2>LightsOut</h2>
+        </a>
+        <input type="search" class="input w-40per" placeholder="Search LightsOut" />
```

Comment:
i think classname `w-40` is good than `w-40per`

Reply:
I have different units for utility classes for width for eg w-40per, w-30vw, w-40rem that's why